sycophancy_analysis/dashboard/app.py

```
@@ -0,0 +1,894 @@
+
+final_loader = FinalResultsLoader(DEFAULT_FINAL_RESULTS_DIR)
+
+
+def _get_final_loader():
+    """Return a FinalResultsLoader based on optional ?prefix= path; fall back to the default loader.
+
+    Allows dashboard consumers to point at any results subfolder without changing server code.
+    """
+    try:
+        prefix = request.args.get('prefix', default=None, type=str)
+    except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)